File: crates/deslop/tests/fixtures/dart-small/beta.dart
int run(int limit) {
  if (limit < 0) {
    return 0;
  }
  var accumulator = 0;
  for (var position = 0; position < limit; position = position + 1) {
    accumulator = accumulator + position;
  }
  return accumulator;
}

File: crates/deslop/tests/fixtures/php-small/beta.php
<?php

function run(int $limit): int
{
    if ($limit < 0) {
        return 0;
    }
    $accumulator = 0;
    for ($position = 0; $position < $limit; $position++) {
        $accumulator = $accumulator + $position;
    }
    return $accumulator;
}
